internal/store/peering_connections.go

```diff
@@ -25,16 +25,16 @@ import (
 //go:generate mockgen -destination=../mocks/mock_peering_connections.go -package=mocks github.com/mongodb/mongodb-atlas-cli/internal/store PeeringConnectionLister,PeeringConnectionDescriber,PeeringConnectionDeleter,AzurePeeringConnectionCreator,AWSPeeringConnectionCreator,GCPPeeringConnectionCreator,PeeringConnectionCreator
 
 type PeeringConnectionLister interface {
-	PeeringConnections(string, *atlas.ContainersListOptions) ([]atlas.Peer, error)
+	PeeringConnections(string, *atlas.ContainersListOptions) ([]interface{}, error)
 }
 
 type PeeringConnectionDescriber interface {
-	PeeringConnection(string, string) (*atlas.Peer, error)
+	PeeringConnection(string, string) (interface{}, error)
 }
 
 type PeeringConnectionCreator interface {
 	CreateContainer(string, *atlasv2.CloudProviderContainer) (interface{}, error)
-	CreatePeeringConnection(string, *atlas.Peer) (*atlas.Peer, error)
+	CreatePeeringConnection(string, *atlasv2.ContainerPeerViewRequest) (interface{}, error)
 }
 
 type AzurePeeringConnectionCreator interface {
```

Comment:
We can consider to add specific methods for retrieving and creating networking peering connections based on the cloud provider, i.e., these interfaces can specific methods like `CreateAzurePeeringConnection`